opentelemetry-sdk/tests/metrics/exponential_histogram/test_exponential_bucket_histogram_aggregation.py

```diff
@@ -1043,14 +1043,16 @@ def collect_and_validate(values, histogram) -> None:
         # run this test case with the same values used in a previous execution,
         # check the value printed by that previous execution of this test case
         # and use the same value for the seed variable in the line below.
-        # seed = 4539544373807492135
+        # seed = 3373389994391084876
 
         random_generator = Random(seed)
         print(f"seed for {currentframe().f_code.co_name} is {seed}")
 
         values = []
         for i in range(2000):
             value = random_generator.randint(0, 1000)
+            while i == 0 and value == 0:
+                value = random_generator.randint(0, 1000)
```

Comment:
```suggestion
            # avoid both values being 0
            value = random_generator.randint(0 if i else 1, 1000)
```

Maybe something like this instead?